utilities/src/main/java/io/syndesis/qe/templates/WildFlyTemplate.java

```diff
@@ -12,20 +12,20 @@
 @Slf4j
 public class WildFlyTemplate {
 
-    public static void deploy(String gitURL, String appName) {
+    public static void deploy(String gitURL, String appName, String branchName) {
         if (!TestUtils.isDcDeployed(appName)) {
-//            Template template;
-//            try (InputStream is = ClassLoader.getSystemResourceAsStream("templates/syndesis-wildfly.yml")) {
-//                template = OpenShiftUtils.getInstance().templates().load(is).get();
-//            } catch (IOException ex) {
-//                throw new IllegalArgumentException("Unable to read template ", ex);
-//            }
-//
-//            Map<String, String> templateParams = new HashMap<>();
-//            templateParams.put("GITHUB_REPO", gitURL);
-//            templateParams.put("APPLICATION_NAME", appName);
-//
-//            OpenShiftUtils.getInstance().templates().withName(appName).delete();
+            //            Template template;
```

Comment:
Can we remove the commented code?

Reply:
I'm not sure, @avano was changing some stuff because of OCP4 and called it `OCP4HACK`. So not sure if it could revert back to the original impl or if it will stay with the hack.

Reply:
@asmigala @jsafarik latest kubernetes client doesn't support ocp 4 yet, so it is sending the request to the wrong endpoint when using ocp4, so ideally it will be commented out later with the client version bump

Reply:
So with that, can this change request be closed? And possibly this PR merged? There are test failures on jenkins which should be resolved with this PR.